src/sonic-yang-models/tests/yang_model_tests/yangTest.json

```diff
@@ -1300,267 +1326,419 @@
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet27": {
 				"alias": "Eth7/4",
 				"lanes": "44",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet28": {
 				"alias": "Eth8/1",
 				"lanes": "45",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet29": {
 				"alias": "Eth8/2",
 				"lanes": "46",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet30": {
 				"alias": "Eth8/3",
 				"lanes": "47",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet31": {
 				"alias": "Eth8/4",
 				"lanes": "48",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet32": {
 				"alias": "Eth9/1",
 				"lanes": "49",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet33": {
 				"alias": "Eth9/2",
 				"lanes": "50",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet34": {
 				"alias": "Eth9/3",
 				"lanes": "51",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet35": {
 				"alias": "Eth9/4",
 				"lanes": "52",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
+				"adv_speeds": "10000,25000",
+				"adv_interface_types": "CR,CR2",
+				"interface_type": "CR"
 			},
 			"Ethernet36": {
 				"alias": "Eth10/1",
 				"lanes": "53",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
```

Comment:
Same here we can remove these fields everywhere, if they are not mandatory thanks.